app.test.js

```diff
@@ -2,11 +2,128 @@ import "@babel/polyfill";
 import request from 'supertest'
 import app from './app'
 
+const environment = process.env.NODE_ENV || 'development'
+const configuration = require('./knexfile')[environment]
+const database = require('knex')(configuration)
+
 describe('Server', () => {
   describe('init', () => {
     it('should return a 200 status', async () => {
       const res = await request(app).get('/')
       expect(res.status).toBe(200)
     });
   });
-});
\ No newline at end of file
+});
+
+describe('GET /api/v1/movies', () => {
+  it('should return a 200 and all of the movies', async () => {
+    const expectedMovies = await database('movies').select();
+    const response = await request(app).get('/api/v1/movies');
+    const movies = response.body;
+
+    expect(response.status).toBe(200);
+    expect(movies).toEqual(expectedMovies);
+  })
+  //create sad path test
+})
+
+describe('GET /api/v1/movies/:genre', () => {
+  it('should return a 200 and all movies matching a specific genre', async () => {
+    const expectedMovies = await database('movies').where('genre', 'action').select();
+    const response = await request(app).get('/api/v1/movies/action');
+    const result = response.body;
+
+    expect(response.status).toBe(200);
+    expect(result).toEqual(expectedMovies);
+  })
```

Comment:
I think our idea of returning a standard set or movies for the user to choose from on the front-end keeps our database requests to a minimum, and is ultimately a good thing. If we for some reason decide that this logic should live on the backend, we can revisit these tests to account for that.